geniza/corpus/admin.py

```diff
@@ -143,6 +142,34 @@ def get_queryset(self, request):
             .annotate(shelfmk_all=ArrayAgg('textblock__fragment__shelfmark')) \
             .order_by('shelfmk_all')
 
+    def change_view(self, request, **kwargs):
```

Comment:
I'm a little surprised you wrote a method for this, I was just imagining using the instance in the view to display information `log_entries.last()` and `log_entries.first()`. (although I guess I didn't say that! sorry)

Reply:
Part of the reason I resorted to this was because I couldn't access the object in context for some reason. I assumed it'd be `object`, which didn't work, tried a few other things, and gave up when I found docs describing this method but nothing else. Advice?

Reply:
I'm not sure where exactly it's documented explicitly, but you can see from [this example](https://docs.djangoproject.com/en/3.1/ref/contrib/admin/#overriding-vs-replacing-an-admin-template) or from my fragment template that the context variable is `original`.